test/torchaudio_unittest/functional/consistency_cpu_test.py

```diff
@@ -0,0 +1,14 @@
+import torch
+
+from torchaudio_unittest import common_utils
```

Comment:
Can you change the import to

```python
from torchaudio_unittest.common_utils import PytorchTestCase
```
Same goes to `cuda_test.py`.

Reply:
no problem, I will do the both~